recipes/urwid/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.3.1" %}
+
+package:
+  name: urwid
+  version: "{{ version }}"
+
+source:
+  fn: urwid-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/u/urwid/urwid-{{ version }}.tar.gz
+  md5: 2e1a005cb31368fe21bfeba2d6ad5a5c
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
We have also been folding one line scripts into the `build` section. If the package were using `distutils` only, it would look like this.

    script: python setup.py install

However, as it uses `setuptools`, we add this.

    script: python setup.py install --single-version-externally-managed --record record.txt

The additional arguments stop `setuptools` from handling some stuff for us that causes `conda-build` problems.